README.md

```diff
@@ -45,10 +34,9 @@ You can open example projects for [JS](examples/example-js), [JVM](examples/exam
 * [Instructions for old versions under Kotlin 1.2 and migration guide](docs/old12.md)
 
 
-## Quick example
+## Quick start
```

Comment:
Are you sure that 'start' is the correct word? This section does not mention any ways to include/download the library, and without it, it's rather hard to start exploring library...